proto/signer.proto

```diff
@@ -22,9 +22,6 @@ service KeyManagement {
 service Signer {
   // Sign calculates a cryptographic signature using the Key associated with a KeyID and returns the signature
   rpc Sign(SignatureRequest) returns (Signature) {}
-
-  // CheckHealth returns the HealthStatus with the service
-  rpc CheckHealth(Void) returns (HealthStatus) {}
```

Comment:
Makes sense, I agree.  It might be worth noting somewhere that the signer's healthcheck is hitting the `KeyManagement` interface level in case we expand on the signer in the future